_posts/0000-01-02-Rrezza.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+This is where the magic might happen :)
```

Comment:
This is where we shall modify the idea. Going to try making a fraction, to test out latex!